web/src/components/molecules/MyIntegration/MyIntegrationContent/index.tsx

```diff
@@ -0,0 +1,46 @@
+import styled from "@emotion/styled";
+import { useNavigate, useParams } from "react-router-dom";
+
+import Tabs from "@reearth-cms/components/atoms/Tabs";
+import MyIntegrationForm from "@reearth-cms/components/molecules/MyIntegration/MyIntegrationForm";
+import MyIntegrationHeader from "@reearth-cms/components/molecules/MyIntegration/MyIntegrationHeader";
+import WebhookForm from "@reearth-cms/components/molecules/MyIntegration/WebhookForm";
+import WebhookList from "@reearth-cms/components/molecules/MyIntegration/WebhookList";
+
+const MyIntegrationContent: React.FC = () => {
+  const { tab, workspaceId, integrationId } = useParams();
+  const isFormEdit = location.pathname.includes("/webhooks/edit");
```

Comment:
This potentially should be moved to the organism as well. But I feel like there is a better way to do this. 
In App.tsx you are using `.../:tab`, but that doesn't feel good. Is here the only place you need to know the tab? If so, I think you can simplify the paths in App.tsx, and just pass a prop in the Page component through to here.

Reply:
Fixed in hooks PR (all moved and fixed), the only occurrence for navigation is in the organism.
https://github.com/reearth/reearth-cms/blob/d3cce3ffd47676971d846ecb934008589b8df638/web/src/components/organisms/Settings/MyIntegrationDetails/index.tsx#L8-L27

Reply:
You've addressed the first part of this comment, but not the second part about the route